### PR TITLE
Task/iotagents plugin

### PR DESCRIPTION
**Pull request by dmoranj**
Add IOTAgent plugin tests. There was no specific IOTAgent plugin created, as the generic REST one fit perfectly. 

Also, refactor Perseo's plugin to create a generic "URL-Table plugin" that checks an assignment from (method, URL) to action.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
 - FIX Service not found for tokens coming from a trust.
-- FIX XAuth Token not checked if validation is off (#197)
\ No newline at end of file
+- FIX XAuth Token not checked if validation is off (#197)
+- ADD Tests for the IOTAgent plugin (using the generic REST plugin).
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Strictly speaking probably this line shouldn't needed, as tests are part of the "developers tool" and this changelog is mostly intented for final users.

However, it is better to "fail" by exceed than by default, so it's ok with the new line.

NTC

File: test/unit/iotagent_plugin-test.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+describe('IOT Agent Plugin tests', function() {
+    var proxy,
+        mockServer,
+        mockApp,
+        mockAccess,
+        mockAccessApp,
+        mockOAuth,
+        mockOAuthApp,
+        apiCases = [
+            ['POST', '/iot/devices', 'create'],
```
Comment (fgalan) [inline test/unit/iotagent_plugin-test.js]: After F2F discussion, the behaviour is not correct... Possible solutions:

* Having separated actions, "createServices", "createDevices" (thus the generic REST pluging could be use)
* Include the resource from URL in the information passed to the AC, thus the AC takes decissions based on action + resource

Comment (dmoranj) [inline test/unit/iotagent_plugin-test.js]: Fixed in 32d4cd2

Using the inclusion of information about the resource.

**Comment by fgalan** [comment]
LGTM

**Pull request merged**